package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A standalone version of the readability library used for Firefox Reader View.",
   "main": "Readability.js",
   "scripts": {
-    "test": "mocha test/index.js",
+    "test": "mocha test/index.js test/test-jsdomparser.js",
```

Comment:
... or prepend test- ? I guess it doesn't matter much, but either will mean moving index.js...